src/main/webapp/get-started.html

```diff
@@ -0,0 +1,308 @@
+watch                               wc                                  while
+            </code></pre>
+            <p>You can then grab more specific help for a given command using the <code>--help</code> option for this command:</p>
+            <pre class="alert alert-primary"><code>
+karaf@root()> bundle:list --help
+DESCRIPTION
+        bundle:list
+
+        Lists all installed bundles.
+
+SYNTAX
+        bundle:list [options] [ids]
+
+ARGUMENTS
+        ids
+                The list of bundle (identified by IDs or name or name/version) separated by whitespaces
+
+OPTIONS
+        -name, -n
+                Show bundle name
+        -u
+                Shows the update locations
+        -r
+                Shows the bundle revisions
+        --no-ellipsis
+
+        -l
+                Show the locations
+        -s
+                Shows the symbolic name
+        --context, -c
+                Use the given bundle context
+                (defaults to 0)
+        --help
+                Display this help message
+        -t
+                Specifies the bundle threshold; bundles with a start-level less than this value will not get printed out.
+        --no-format
+                Disable table rendered output
+            </code></pre>
+            <p>
+              Note that the console supports tab completion so if you start typing a command it will show all possible completions and also auto
+              complete if there is only one completion.
+            </p>
+          </div>
+        </div>
+
+        <div class="row mb-2 mt-2">
+          <div class="col">
+            <h4 class="pb-3 mb-1">Stop</h4>
+            You have multiple options for shuting down your Karaf instance:
+            <ul>
+              <li>For a background running instance, run the command:
+                <pre class="alert alert-primary"><code>$KARAF_HOME/bin/stop</code></pre>
+              </li>
+              <li>To stop Karaf from the console, enter <code>Ctrl+D</code> in the console.</li>
+              <li>Alternatively, you can also run the following command:
+                <pre class="alert alert-primary"><code>
+karaf@root()> feature:install system
+karaf@root()> system:shutdown
+Confirm: halt instance root (yes/no): yes
+karaf@root()>
+                </code></pre>
+              </li>
+            </ul>
+            <p><i class="fa fa-info-circle"></i> Halt is also an alias for <code>system:shutdown</code>
+              <pre class="alert alert-primary"><code>karaf@root()> halt</code></pre>
+            </p>
+          </div>
+        </div>
+
+        <!-- Easy to develop -->
+        <div class="row mb-2 mt-3">
+          <div class="col">
+            <h2 class="pb-3 mb-4 font-italic border-bottom"><i class="fas fa-code"></i> Easy to develop</h2>
+            <p>You are now ready to develop your first bundle!</p>
```

Comment:
Instead of bundle, I would rather use application. OSGi related wording can scare some devs. And Karaf can also work with non OSGi application.

Reply:
done!